ros2_generate_interface_docs/ros2_generate_interface_docs/templates/msg-index.html.em

```diff
@@ -0,0 +1,15 @@
+<html>
+  <head>
+    <title>@(package) Action/Msg/Srv Documentation</title>
```

Comment:
I would suggest the order from most primitive to most complex: Message / Service / Action.